components/actions_menu/actions_menu.scss

```diff
@@ -0,0 +1,35 @@
+.VisitMarketplaceText {
+    margin: 12px;
+    font-family: 'Open Sans';
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 20px;
+    text-align: center;
+}
+
+.VisitMarketplace {
+    top: 1px;
+    padding-top: 4px;
+    padding-right: 16px;
+    padding-bottom: 16px;
+    padding-left: 16px;
+    border-radius: 4px;
+    font-size: 12px;
+    text-align: center;
+}
+
+.VisitMarketplaceButton {
+    border-radius: 6px;
```

Comment:
@jfrerich as noted in my comment, this border radius should be `border-radius: 4px`

Reply:
resolved in a87d771